src/main/resources/application.yml

```diff
@@ -22,7 +22,8 @@ quarkus:
     db-kind: postgresql
   hibernate-orm:
     database:
-      generation: drop-and-create
+      #   We should use Flyway or Liquibase for schema migration
+      generation: update
```

Comment:
agreed. Flyway works quite well in my experience. Example setup here https://github.com/jcechace/debezium-devconf24-demo/tree/main/debezium-crypto-app/src/main